test/http-client-benchmarks/pom.xml

```diff
@@ -175,7 +175,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-shade-plugin</artifactId>
-                <version>2.2</version>
+                <version>3.5.0</version>
```

Comment:
This was bumped because the build was failing with:

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-shade-plugin:2.2:shade (default) on project http-client-benchmarks: Unable to parse configuration of mojo org.apache.maven.plugins:maven-shade-plugin:2.2:shade for parameter Main-Class: Cannot assign configuration entry 'Main-Class' with value 'org.openjdk.jmh.Main' of type java.lang.String to property of type java.util.Map -> [Help 1]
```

It's not entirely clear why the build fails with it, and why these changes cause this since they seem completely unrelated, but we do have mixed versions of the shade plugin across multiple packages, so i just bumped it to the most recent version